### PR TITLE
#185218923 A seller should be able to delete a specific item from their collection

### PR DESCRIPTION
**Pull request by ReneNtabana**
## What does this PR do?
Delete a product from the seller's collection
## Description of Task to be completed?
A seller should be able to delete a specific item from their collection.
## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

[#185218923](https://www.pivotaltracker.com/story/show/185218923)
## Screenshots (if appropriate)
Screenshot (13)
Screenshot (14)

### CODE REVIEW

**Review by jcharles921 — APPROVED** [review]
LGTM !!! 👌🏽

**Review by cyubahiro367 — APPROVED** [review]

File: src/styles/index.scss
Comment (bergad37) [inline src/styles/index.scss:1]: follow naming convetion, use lowercase for naming `Dialog scss` file !!

File: src/containers/SellerProductsContainer.jsx
Comment (bergad37) [inline src/containers/SellerProductsContainer.jsx:1]: Remove this spacing ,  line 52-54!

**Review by mken1205 — APPROVED** [review]

File: src/styles/index.scss
Comment (ReneNtabana) [reply src/styles/index.scss:1]: Done!

File: src/containers/SellerProductsContainer.jsx
Comment (ReneNtabana) [reply src/containers/SellerProductsContainer.jsx:1]: Done

**Review by irakevine — APPROVED** [review]
Great Work! @ReneNtabana

**Review by bergad37 — APPROVED** [review]
Well done!

File: src/containers/SellerProductsContainer.jsx [hunk 1]
```diff
@@ -140,23 +203,14 @@ const SellerProductsContainer = () => {
...
-              <div
-                key={product.id}
-                className='sellingProd__card flex  mb-5 '
-                onClick={() => {
-                  return getProduct(index);
-                }}
-              >
+              <div key={product.id} className='sellingProd__card flex  mb-5 '
+              onClick={() => {
+                // document.querySelector('#one').classList.add('bg-black','bg-opacity-50');
```
Comment (nishimweprince) [inline src/containers/SellerProductsContainer.jsx]: Please remove these commented lines of code.

**Review by nishimweprince — APPROVED** [review]
This is excellent. I, however, added a comment to remove a commented line, please implement it.

File: src/containers/SellerProductsContainer.jsx (hunk 1 above)
Comment (ReneNtabana) [reply src/containers/SellerProductsContainer.jsx]: I've resolved the issue

**Review by niyontwali — APPROVED** [review]

**Pull request merged**